### ISSUE TITLE
Is it possible to have an aggregate function instead of the active binding?

### ISSUE THREAD

**Issue by schalkdaniel** [mlr-org/mlr3#153]
Wouldn't it be nice to have a function
```
bmr$aggregate(params = TRUE)
```
which returns not just the aggregated benchmark object but also the parameter of a single learner? I have written a wrapper in tuning, but it feels a bit misplaced there.

**Comment by mllg** [comment]
This is now a function, as requested. The interface might change again though, depending on #155.

**Comment by mllg** [comment]
Actually, it might be better to provide the parameters as list column and let the user unnest them. @schalkdaniel What do you think?

**Comment by schalkdaniel** [comment]
I think the `data.table` structure with `R6` class columns is complicated enough. Unnesting the parameters manually requires to know `mlr3misc` and from there the `unnest` function. Providing some help here is ok (in my opinion).

On the other hand, tuning over a large amount of parameter (as it is for pipelines?) can bloat the aggregated data table.

So, maybe returning the parameters as list column by default, but giving the possibility to unnest them automatically?

**Comment by mllg** [comment]
We have a no sugar agreement. You get the list column for now. :wink: 

I've added an example to demonstrate how to access the params.
